lib/topology_lib_ip/library.py

```diff
@@ -194,11 +318,40 @@ def add_link_type_vlan(enode, portlbl, name, vlan_id, shell=None):
     response = enode(cmd, shell=shell)
     assert not response
 
+from pytest import set_trace
+
+def show_interface(enode, dev):
```

Comment:
Add ``shell`` argument here, default it to ``None`` as in the rest of the library functions.